packages/contracts/src/Semaphore.sol

```diff
@@ -0,0 +1,243 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { ISemaphore } from "@semaphore-protocol/contracts/interfaces/ISemaphore.sol";
+import { ISemaphoreVerifier } from "@semaphore-protocol/contracts/interfaces/ISemaphoreVerifier.sol";
+import { SemaphoreGroups } from "@semaphore-protocol/contracts/base/SemaphoreGroups.sol";
+import { MIN_DEPTH, MAX_DEPTH } from "@semaphore-protocol/contracts/base/Constants.sol";
+import { HatsExcubia } from "@zk-kit/excubiae/extensions/HatsExcubia.sol";
+
+/// @title Semaphore
+/// @dev A minimal version of the Semaphore contract.
+contract Semaphore is ISemaphore, SemaphoreGroups {
```

Comment:
is this pattern better than extending Semaphore and overriding the functions we changed?

Reply:
`Semaphore.sol` functions are not virtual and we are just taking Semaphore and extrapolating what we need to handle one group. This should cost less gas and be resilient enough for the sake of the hackathon.

Reply:
perfect